web/libs/editor/src/components/BottomBar/CurrentTask.jsx

```diff
@@ -39,23 +74,38 @@ export const CurrentTask = observer(({ store }) => {
             ))}
         </div>
         {historyEnabled && (
-          <div className={cn("current-task").elem("history-controls").toClassName()}>
-            <Button
-              variant="neutral"
-              data-testid="prev-task"
-              disabled={!historyEnabled || !store.canGoPrevTask}
-              onClick={store.prevTask}
-            >
-              <IconChevronLeft />
-            </Button>
-            <Button
-              data-testid="next-task"
-              disabled={!store.canGoNextTask && !canPostpone}
-              onClick={store.canGoNextTask ? store.nextTask : store.postponeTask}
-              variant={!store.canGoNextTask && canPostpone ? "primary" : "neutral"}
-            >
-              <IconChevronRight />
-            </Button>
+          <div
+            className={cn("current-task")
+              .elem("history-controls")
+              .mod({ newui: isFF(FF_DEV_3873) })
```

Comment:
@yyassi-heartex Do you recall what this `FF_DEV_3873` is for? I feel there are a lot of changes in this PR that are hinging on its lingering existence, which makes it hard to understand what is necessary.

Reply:
@bmartel we are basically on UI v3 right now. We had v1 which was finally removed some time ago (FF_1170). And we still have v2 hidden behind feature flag. It's enabled for everyone and is ready to be removed, but it's everywhere, including some tests, so we have to fix them first. One day, one day...